backoffice/src/controllers/auth/index.js

```diff
@@ -0,0 +1,67 @@
+import axios from 'axios';
+
+/**
+ * Call yakchat auth server
+ */
+export function login (username, password) {
+	axios.post('http://localhost:8080/api/api/v1/auth/login', 
+		JSON.stringify({ username, password }), 
+		{headers: {
+			'Accept': 'application/json',
+			'Content-Type': 'application/json'
+		}}
+	)
+	.then((response) => {
+		console.log(response);
+		localStorage.setItem('udata', JSON.stringify({
+			accessLevel: 5,
+			admin: false,
+			uid: '0FxY4aLtrnhRaIzQAbmNNzyarN72',
+			displayName: 'GithubClient',
+			email: 'gh-client-t01@rebeldemo.com',
+			emailVerified: true
+		}))
+		if (process.env.ENVIROMENT === 'PRODUCTION') {
+			document.location.pathname = '/backoffice.html';
```

Comment:
If we use navigo or navigo is installed why are we handling the routes on this way ?

Reply:
removed from the package.json